.github/workflows/run_djlint.yml

```diff
@@ -3,7 +3,11 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'frontend/templates/**'
```

Comment:
Would it not be better to check for `*.html` since djlint will run on all html files anyways

Reply:
fixed